{
  if (throwable != null) {
    throw new RuntimeException("An error was encountered",throwable);
  }
  return result;
}
